consensus/core/src/block_validator.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::block::Block;
+use async_trait::async_trait;
+
+/// The interfaces to validate the legitimacy of a statement block's contents.
+#[async_trait]
+pub trait BlockValidator: Send + Sync + 'static {
+    type Error: std::fmt::Display + std::fmt::Debug + Send + Sync + 'static;
+    /// Determines if a statement block's content is valid.
+    async fn validate(&self, _b: &Block) -> Result<(), Self::Error>;
+
+    async fn validate_all(&self, _b: &[Block]) -> Result<(), Self::Error>;
+}
+
+#[derive(Clone)]
+pub struct AcceptAllBlockValidator;
```

Comment:
What is this used for? Is it just for the test? If we are actually going to have a `BlockValidator` that does not do any checks maybe we can call it as `NoCheckBlockValidator` or if it is just for testing we can name it as `TestBlockValidator`

Reply:
We can go with `TestBlockValidator`